ddtrace/tracer/option.go

```diff
@@ -521,6 +538,14 @@ func WithPropagator(p Propagator) StartOption {
 	}
 }
 
+// WithServiceNamePropagation enables the hashed name of this service to be horizontally
+// propagated to other services (e.g. over http requests).
```

Comment:
```suggestion
// propagated to other services (e.g. over HTTP requests).
```
What feature does this propagation enable? Why would a user want to set this option?

Reply:
👍 Added a note about the new ingestion control page, let me know if you think that makes it clear enough 😃